app/backend/approaches/retrievethenread.py

```diff
@@ -9,10 +8,6 @@
 from core.authentication import AuthenticationHelper
 from core.messagebuilder import MessageBuilder
 
-# Replace these with your own values, either in environment variables or directly here
```

Comment:
This was added in the GPT4-vision merge. I'm guessing it was added during development and never removed. We set up the storage accounts in app.py now.